henchman/errors_test.go

```diff
@@ -0,0 +1,24 @@
+package henchman
+
+import (
+	"fmt"
+	log "gopkg.in/Sirupsen/logrus.v0"
```

Comment:
import logrus only in the main package. Abstract it out so that in case we switch the package, we need to change it in only one place